### PR TITLE
Update LRANGE tests for Redis 6

### PR DESCRIPTION
**Pull request by sazzad16**

### CODE REVIEW

**Comment by sazzad16** [comment]
@gkorland @marcosnils ping!

File: src/test/java/redis/clients/jedis/tests/commands/ListCommandsTest.java [hunk 1]
```diff
@@ -129,9 +128,7 @@ public void lrange() {
     range = jedis.lrange("foo", 1, 2);
     assertEquals(expected, range);
 
-    expected = new ArrayList<String>();
-    range = jedis.lrange("foo", 2, 1);
-    assertEquals(expected, range);
+    assertNull(jedis.lrange("foo", 2, 1));
```
Comment (gkorland) [inline src/test/java/redis/clients/jedis/tests/commands/ListCommandsTest.java]: is that a backward issue in Redis 6?

Comment (sazzad16) [reply src/test/java/redis/clients/jedis/tests/commands/ListCommandsTest.java]: Yes, a backward issue in Redis 6.
PS: Not a backward issue for Jedis.

Comment (gkorland) [reply src/test/java/redis/clients/jedis/tests/commands/ListCommandsTest.java]: I think it should be fixed before 6.x will be released

**Pull request merged**